.github/CODEOWNERS

```diff
@@ -29,6 +29,7 @@ kms @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/java-samples-reviewers @
 orgpolicy @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver
 privateca @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver
 recaptcha_enterprise @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver
+recaptcha_enterprise/demosite @Sita04 @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/recaptcha-customer-obsession-reviewers
```

Comment:
Any chance you can change this to a group your a member of?  We prefer not to have individual users here.

@averikitsch FYI

Reply:
Oh, not a problem. I am part of the dee-infra group anyway. Thanks!